sdk/src/client.ts

```diff
@@ -28,44 +28,58 @@ import { validateAllowlist, validateMetaData, verifyMerkleProof, verifyMerklePro
  * @param storage - Hypercerts storage object
  */
 export default class HypercertClient implements HypercertClientInterface {
-  readonly _config: HypercertClientConfig;
+  private _config: HypercertClientConfig;
```

Comment:
Does setting this as private instead of readonly again, not break the `Config` component in the frontend?

Reply:
Ah, I see it is because the config is being set dynamically on line 64. However, I think we need to expose it for the frontend multi chain functionality to work? Probably by adding some sort of `getClientConfig()` function on the base class, that can be called from the frontend.

Reply:
There's already a config getter in this class?